src/PowerShellEditorServices/PowerShellEditorServices.csproj

```diff
@@ -39,7 +39,7 @@
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
     <PackageReference Include="Microsoft.Extensions.FileSystemGlobbing" Version="3.1.3" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.3" />
-    <PackageReference Include="OmniSharp.Extensions.LanguageServer" Version="0.16.0" />
+    <PackageReference Include="OmniSharp.Extensions.LanguageServer" Version="0.17.0-*" />
```

Comment:
@TylerLeonhardt @rjmholt 
This change break the build in the last view days because on that myget server, they also publish packages that end with `0.17.0-pullrequest`. We should use something like `0.17.0-alpha*` instead.